src/components/views/JobInfo.js

```diff
@@ -136,7 +135,7 @@ export default class JobInfo extends React.Component {
               <Col xs={4}>
                 <KeyValue label={<FormattedMessage id="ui-local-kb-admin.prop.runningStatus" />}>
                   <div data-test-job-status>
-                    {get(job, 'status.label', '-')}
+                    {job?.status?.label ?? '-'}
```

Comment:
Use the stripes-components' `<NoValue>` component here instead of rendering a `'-'` directly.

Reply:
Have done for several similar places in this file